### DISCUSSION TITLE
Aqara Temperature unfortunately?

### DISCUSSION THREAD

**Discussion by 3PrintD-Solution** [Koenkk/zigbee2mqtt#9174] [Devices]
Hello Koenkk

I have some Aqara temperature, humidity and pressure Sensor. 
In Zigbee2MQTT the temperatures are unfortunately after two commas
displayed. Is that what you wanted?

`"temperature": 22.27,`

you cannot output this to 

`"temperature": 22.3,`

hope it is possible

**Comment by Danit2** [comment]
https://www.zigbee2mqtt.io/devices/WSDCGQ11LM.html#device-type-specific-configuration
